### DISCUSSION TITLE
Bigquery connection issues

### DISCUSSION THREAD

**Discussion by jv1522** [cloudera/hue#3091] [Questions] [answered]
I was using hue app to connect to google cloud bigquery which was working a week ago. Without any change in the application, the app now throws bigquery permission issue with the below error message: 

Testing the connector connection failed. {'status': 1, 'message': '403 POST https://bigquery.googleapis.com/bigquery/v2/projects/prj-df-metap...

Is there a way to debug this to find out the root cause?

**Comment by Harshg999** [comment, marked as answer]
Hey @jv1522, are you seeing this issue in recent Hue docker images? We recently removed shipping `pybigquery` package in Hue docker image because of `protobuf` version issue.

If this is related, then you can try installing `pybigquery` and see if that helps.

**Comment by jv1522** [comment]
Hi @Harshg999 it resolved after I included pybigquery.
By the way, this might not be related to this issue, but do we support oauth 2.0 if we want to enable google login in hue app?

**Comment by Harshg999** [comment]
I believe we have some implementation to support OAuth but that is very old. I am not sure currently if we have support for oauth 2.0
However doing a quick search in the codebase, I can see a lot of references to oauth configs under `[[oauth]]` or `[[liboauth]]`.
